shared/profile/container.js

```diff
@@ -116,23 +116,15 @@ const mapDispatchToProps = (dispatch, {setRouteState}: OwnProps) => ({
         [peopleTab]
       )
     ),
-  _onSendOrRequestLumens: (to: string) => {
-    dispatch(WalletsGen.createClearBuildingPayment())
-    dispatch(WalletsGen.createClearBuiltPayment())
+  _onSendOrRequestLumens: (to: string, isRequest) => {
+    dispatch(WalletsGen.createClearBuilding())
```

Comment:
Sometimes when hitting send/request from profile, the user in the 'to' field won't match the one I came from.
1. Go to ayoubd -> hit request -> 'ayoubd' in to field
2. In followers click mikem -> hit request -> 'mikem' in to field
3. Hit back -> hit request -> 'mikem' in to field, but we were on ayoubd's profile

Reply:
Ah! ouch. good find.